api/controllers/file_processor/file-base64.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * file_processor/file-base64.js
+ *
+ *
+ * @api {get} /file/:ID/base64 Get a File as base64
+ * @apiGroup File
+ * @apiPermission User
+ * @apiParam {string} ID file uuid
+ * @apiQuery {boolean} mobile whether to reutrn downscaled images for mobile
+ * @apiUse successRes
+ * @apiSuccess (200) {Object} data
+ * @apiSuccess (200) {string} data.image base64
+ */
+
+var inputParams = {
+   ID: { string: { uuid: true }, required: true },
+   mobile: { boolean: true },
```

Comment:
we should probably either specify `required` or `optional` here.

Reply:
good catch, yes meant it to be optional